docs/source/concept_models/vivarium_swissre_cervicalcancer/cervical_cancer_concept_model.rst

```diff
@@ -0,0 +1,597 @@
+
+  ==============
+  Document Title
+  ==============
+
+  Section Level 1 (#.0)
+  +++++++++++++++++++++
+  
+  Section Level 2 (#.#)
+  ---------------------
+
+  Section Level 3 (#.#.#)
+  ~~~~~~~~~~~~~~~~~~~~~~~
+
+  Section Level 4
+  ^^^^^^^^^^^^^^^
+
+  Section Level 5
+  '''''''''''''''
+
+  The depth of each section level is determined by the order in which each
+  decorator is encountered below. If you need an even deeper section level, just
+  choose a new decorator symbol from the list here:
+  https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#sections
+  And then add it to the list of decorators above.
+
+
+.. _cervical_cancer_concept_model:
+
+=======================================
+Vivarium CSU Cervical Cancer Simulation
+=======================================
+
+.. contents::
+  :local:
+
+.. list-table:: List of abbreviations
+   :header-rows: 1
+
+   * - Label
+     - Definition
+   * - BCC
+     - benign cervical cancer
+   * - CC
+     - cervical cancer 
+   * - ACMR
+     - all-cause mortality rate
+   * - prev_c432
+     - prevalence of cervical cancer
+   * - incidence_c432
+     - incidence of cervical cancer
+   * - csmr_c432
+     - cause-specific mortality rate of cervical cancer
+
+.. _1.0:
+
+1.0 Background
+++++++++++++++
+
+.. _1.1:
+
+1.1 Project overview
+--------------------
+This project will generate forecasts of cervical cancer mortality and morbidity 
+to allow Swiss Re to identify trends that are important to its business 
+decision-making. IHME will produce both a baseline (business as usual) forecast, 
+and an alternative scenario forecast in which key cervical cancer screening 
+practice and HPV vaccination are implemented in a simulation framework. Baseline 
+forecasts will incorporate expected trends in relevant risk factors. Alternative 
+scenario forecasts will incorporate baseline forecasts and the expected impact of 
+new screening technologies and vaccination. All forecasts will represent the Swiss 
+Re’s insured population from the weighted blend of Chinese provinces. Forecasts 
+run from year 2020 to 2040.
+
+.. _1.2:
+
+1.2 Literature review
+---------------------
+
+.. todo::
+
+ add more literature background
+
+
+.. _2.0:
+
+2.0 Modeling aims and objectives
+++++++++++++++++++++++++++++++++
+IHME will estimate the yearly number of cases of benign and invasive cervical 
+cancer detected under specific cervical cancer screening practices and the yearly 
+number of deaths from undetected cervical cancer (both in unit of per 100,000 
+insured person-years) in order to identify pay-out trends for cervical cancer 
+claims.
+
+
+.. _3.0:
+
+3.0 Causal framework
+++++++++++++++++++++
+
+.. _3.1:
+
+3.1 Causal variables
+--------------------
+ 
+Outcome
+ - Cervical cancer detection (benign and invasive)
+ - Mortality and morbidity of cervical cancer
+Most proximal determinant/exposure
+ - Cervical cancer status
+ - Cervical cancer screening
+ - HPV infection  
```

Comment:
Include HPV vaccination status here?

Reply:
> Thank you for getting this posted, it is really helpful to have it all together in the repo. Good work! I've made a few comments, and I think we need to develop the HPV / HPV vaccination part of the model some more. But this is very helpful and thanks again for getting the pull request in.

I am working on adding more details to HPV component, both screening strategy and risk related data.